scripts/backup-mongodb.sh

```
@@ -0,0 +1,110 @@
+    
+    # Return the archive path
+    echo "$archive_file"
+}
+
+# Cleanup old archives (older than specified days)
+cleanup_old_archives() {
+    local retention_days="${RETENTION_DAYS:-30}"
+    
+    log "Cleaning up archives older than $retention_days days..."
+    
+    find "$BACKUP_DIR" -name "*.archive.gz" -type f -mtime +$retention_days -delete
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)